src/matter/cluster/server/OperationalCredentialsServer.ts

```diff
@@ -119,7 +119,13 @@ export const OperationalCredentialsClusterHandler: (conf: OperationalCredentials
 
```

Comment:
Removing session associated with a fabric when a fabric is deleted should be done by the FabricManager, otherwise there is too much risk to have the system in an incoherent state.

Reply:
I moved it to "MatterDevcie" now because FabricManager do not have acces to the session controller but the device is taling care here too ... so should work? 

ALternatively Ineed to pass the sessionmanager with removeFabric justto call from there ...